docs/guide/tutorial-i18n.md

```diff
@@ -101,19 +115,47 @@ method to perform the actual translation work. The component can be configured i
 ],
 ```
 
-In the above code, a message source supported by [[yii\i18n\PhpMessageSource]] is being configured. The pattern
-`app*` indicates that all message categories whose names start with `app` should be translated using this
-message source. The [[yii\i18n\PhpMessageSource]] class uses PHP files to store message translations.
-These files contain a simple PHP array that is a map of the messages in source language to the translation in the target language.
-Each PHP file corresponds to the messages of a single category. By default, the file name should be the same as
-the category name. However, you may configure [[yii\i18n\PhpMessageSource::fileMap|fileMap]] to map a category
-to a PHP file with a different naming approach. In the above example, the category `app/error` is mapped to
-the PHP file `@app/messages/ru-RU/error.php` (assuming `ru-RU` is the target language). Without this configuration,
-the category would be mapped to `@app/messages/ru-RU/app/error.php`, instead.
+In the above code, a message source supported by [[yii\i18n\PhpMessageSource]] is being configured. 
+
+#### Category wildcards with \* symbol
```

Comment:
why is this a headline on the same level as the 1. 2. 3. steps? also, a headline for one sentence seems unnecessary.